src/Pages/Signup/index.js

```diff
@@ -1,7 +1,258 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { API } from '../../config';
+import { fetchPost } from '../../utils/fetches';
+import { REGEXP, validate } from '../../utils/regex';
+import './index.scss';
+
+class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      nickname: '',
+      phone_number: '',
+      gender: '',
+      year: '',
+      month: '',
+      day: '',
+    };
+  }
+
+  validateInputData = (id, pw) => {
+    return (
+      validate(id, REGEXP.emailRegExp) && validate(pw, REGEXP.passwordRegExp)
+    );
+  };
+
+  handleInput = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+    console.log(this.state);
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    if (!this.validateInputData(this.state.email, this.state.password)) return;
+    let response = fetchPost(API, {
+      email: this.state.email,
+      password: this.state.password,
+      phone_number: this.state.phone_number,
+      nickname: this.state.nickname,
+      birth: `${this.state.year}.${this.state.month}.${this.state.day}`,
+      gender: this.state.gender,
+    }).then((result) => console.log('결과: ', result));
```

Comment:
- 특별한 이유 없다면 가급적 const로 선언해주시는 것이 좋겠습니다
- 구조분해 할당 후 객체 단축 속성명을 사용하면 좀더 간결하게 코드 작성할 수 있을거 같습니다
```suggestion
    const response = fetchPost(API, {
      email,
      password,
      phone_number,
      nickname,
      gender,
      birth: `${this.state.year}.${this.state.month}.${this.state.day}`,
    }).then((result) => console.log('결과: ', result));
```